lib/actions/dragAndDrop.js

```diff
@@ -15,6 +15,14 @@ const dragAndDrop = async (source, destination, options) => {
     isSelector(destination) || isString(destination) || isElement(destination)
       ? await waitAndGetActionableElement(destination, options.force)
       : destination;
+  if (!(await sourceElem.isVisible())) {
+    throw new Error('Taiko cannot drag hidden elements');
+  }
+  if (isSelector(dest) || isString(dest) || isElement(dest)) {
```

Comment:
isVisible is not applicable of string its enough if we just check isElement since we are already fetching destination element for other cases